frontend/app/src/App.tsx

```diff
@@ -896,6 +898,7 @@ export class App extends PureComponent<Props, State> {
         appPages: newSessionProto.appPages,
         currentPageScriptHash: newPageScriptHash,
         latestRunTime: performance.now(),
+        currentFragmentId: newSessionProto.fragmentId || "",
```

Comment:
nit: should it be `newSessionProto.fragmentId || ""` because we want to test the left-side for `null` or `undefined`?

Reply:
Ah, yeah -- I think using ?? would be slightly nicer here